packages/shared/README.md

```
@@ -0,0 +1,14 @@
+# @realm-walker/shared 📚
+
+The "Truth" of the Realm. This package contains the core Zod schemas, TypeScript types, and universal constants used across all other packages.
+
+## Core Schemas
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)